indico/modules/rb_new/controllers/backend.py

```diff
@@ -42,6 +42,8 @@
 from indico.web.util import jsonify_data
 
 
+NUMBER_OF_SUGGESTIONS = 5
```

Comment:
`NUM_SUGGESTIONS`? "of" is one of these words that usually make variable names overly verbose